<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create distance distribution</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Network file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInAmiraFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelX0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX0">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCentre">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Centre</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditY0">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelZ0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>z0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditZ0">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSphere">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Use spherical subregion?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribution file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRadius">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRadius">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBdryFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New lymphatic bdry file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBdryFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxBdryFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Use existing boundary file?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLymphaticFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lymphatic tiff file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLymphaticFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonZoom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>L_distancer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBdryFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bdryFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxBdryFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>cboxBdryFileToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLymphaticFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>lymphaticFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>radioButtonHandler()</slot>
  <slot>L_distancer()</slot>
  <slot>bdryFileSelecter()</slot>
  <slot>cboxBdryFileToggled()</slot>
  <slot>lymphaticFileSelecter()</slot>
 </slots>
</ui>
